flink-table/flink-sql-client/pom.xml

```diff
@@ -57,13 +61,19 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-table_${scala.binary.version}</artifactId>
+			<artifactId>flink-cep_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
 		</dependency>
 
+		<!-- Table ecosystem -->
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-cep_${scala.binary.version}</artifactId>
+			<artifactId>flink-table-api-java-bridge_${scala.binary.version}</artifactId>
```

Comment:
Same to the previous comments.

Reply:
Logically, the SQL client wants to bridge to DataStream API but also wants to execute queries. It is an internal coincident that the old Flink planner depends on the bridge for getting DataStream related classes.